src/app/post/[id]/page.tsx

```diff
@@ -9,7 +9,7 @@ interface IPostAppPageParams {
 }
 
 const Post = ({ params }: IPostAppPageParams) => {
```

Comment:
이미 post/ 페이지에서 Post라는 이름을 사용하고 있는데 어차피 상관없긴 하지만 이름을 바꾸는게 어떤가요?

```suggestion
 const PostDetail = ({ params }: IPostAppPageParams) => {
```

Reply:
좋습니다!